webpack.config.js

```diff
@@ -105,6 +106,11 @@ const config = {
 		],
 	},
 	plugins: [
+		new DefinePlugin( {
+			// Inject the `GUTENBERG_PHASE` global, used for feature flagging.
+			// eslint-disable-next-line @wordpress/gutenberg-phase
+			'process.env.GUTENBERG_PHASE': JSON.stringify( parseInt( process.env.npm_package_config_GUTENBERG_PHASE, 10 ) || 1 ),
```

Comment:
`JSON.stringify` is often used with `DefinePlugin` because the substitution is _verbatim_, and thus for something like a string, you'd want to be certain you're replacing it with a string, not an identifier token (the difference between `foo( bar );` and `foo( "bar" )`). For a number, though, the verbatim substitution is fine, i.e. there's no difference with or without `JSON.stringify` (it's `foo( 10 )` with or without `JSON.stringify`).